File: application/entities/website.php
<?php
namespace entities;

/**
 * @Entity
 **/
class Website {

    //...
    /**
    * @Id @GeneratedValue @Column(type="integer")
    * @var integer
    */
    private $id = 0;
    //...

    /**
    * @OneToMany(targetEntity="Domain", mappedBy="website")
    */
    private $domains = array();
    
    /**
    * @OneToMany(targetEntity="Template", mappedBy="website")
    */
    private $templates = array();

    /**
    * @OneToMany(targetEntity="Module", mappedBy="website")
    */
    private $modules = array();

    /**
    * @OneToMany(targetEntity="Page", mappedBy="website")
    */
    private $pages = array();

    /**
    * @ManyToOne(targetEntity="Organisation", inversedBy="website")
    */
    private $organisation = 0;
    
    /**
    * @Column(type="string")
    * @var string
    */
    private $created = "";

    /**
    * @Column(type="string")
    * @var string
    */
    private $name = "";

    /**
    * @Column(type="integer")
    * @var integer
    */
    private $template = 0;



    
    public function get_id() {
	return $this->id;
    }

    public function set_id($id) {
	$this->id = $id;
    }

    public function get_domains() {
	return $this->domains;
    }

    public function set_domains($domains) {
	$this->domains = $domains;
    }

    public function get_templates() {
	return $this->templates;
    }

    public function set_templates($templates) {
	$this->templates = $templates;
    }

    public function get_modules() {
	return $this->modules;
    }

    public function set_modules($modules) {
	$this->modules = $modules;
    }

    public function get_pages() {
	return $this->pages;
    }

    public function set_pages($pages) {
	$this->pages = $pages;
    }

    public function get_created() {
	return $this->created;
    }

    public function set_created($created) {
	$this->created = $created;
    }

    public function get_name() {
	return $this->name;
    }

    public function set_name($name) {
	$this->name = $name;
    }
    
    public function get_template() {
	return $this->template;
    }

    public function set_template($template) {
	$this->template = $template;
    }

    public function get_organisation() {
	return $this->organisation;
    }

    public function set_organisation($organisation) {
	$this->organisation = $organisation;
    }


}

File: src/main/java/dk/apaq/shopsystem/entity/Website.java
package dk.apaq.shopsystem.entity;

import java.io.Serializable;
import java.util.ArrayList;
import java.util.List;

import javax.persistence.Entity;
import javax.persistence.OneToMany;

/**
 * Defines a Website.
 */
@Entity
public class Website extends AbstractContentEntity implements Serializable {

    @OneToMany
    private List<Domain> domains = new ArrayList<Domain>();
    
    /*@OneToMany
    private List<Template> templates = new ArrayList();
    @OneToMany
    private List<Module> modules = new ArrayList<Module>();*/
    
    @OneToMany
    private List<Page> pages = new ArrayList<Page>();
    private String name;
    private String tracking_code;
    private String tracking_code_invoice;

    public List<Domain> getDomains() {
        return domains;
    }

    public void setDomains(List<Domain> domains) {
        this.domains = domains;
    }

    /*public List<Module> getModules() {
        return modules;
    }

    public void setModules(List<Module> modules) {
        this.modules = modules;
    }*/

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public List<Page> getPages() {
        return pages;
    }

    public void setPages(List<Page> pages) {
        this.pages = pages;
    }

    /*public List<Template> getTemplates() {
        return templates;
    }

    public void setTemplates(List<Template> templates) {
        this.templates = templates;
    }*/

    public String getTracking_code() {
        return tracking_code;
    }

    public void setTracking_code(String tracking_code) {
        this.tracking_code = tracking_code;
    }

    public String getTracking_code_invoice() {
        return tracking_code_invoice;
    }

    public void setTracking_code_invoice(String tracking_code_invoice) {
        this.tracking_code_invoice = tracking_code_invoice;
    }
}
